source/scpd/quorum/QuorumTracker.d

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+
+    Bindings for quorum/QuorumTracker.h
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module scpd.quorum.QuorumTracker;
+
+import scpd.Cpp;
+import scpd.scp.SCP;
+import scpd.types.Stellar_SCP;
+import scpd.types.Stellar_types;
+
+extern (C++, `stellar`):
+
+// helper class to help track the overall quorum over time
```

Comment:
We could improve this documentation. It was written by Stellar devs, but it's hard to parse.